packages/@ember/-internals/glimmer/lib/utils/bindings.ts

```diff
@@ -128,11 +129,24 @@ class StyleBindingReference extends CachedReference<string | SafeString> {
 
   compute(): string | SafeString {
     let value = this.inner.value();
-    let isVisible = this.isVisible.value();
 
-    if (isVisible !== false) {
-      return value;
-    } else if (!value) {
+    if (EMBER_COMPONENT_IS_VISIBLE) {
+      let isVisible = this.isVisible.value();
+
+      if (isVisible !== undefined) {
+        deprecate(`\`isVisible\` is deprecated (from "${this.isVisible.parentValue._debugContainerKey}")`, false, {
+          id: 'ember-component.is-visible',
```

Comment:
Is seems that `parentValue` isn't exposed as part of `VersionedReference`, I'm not sure of another way to reference the component instance from here though

<img width="965" alt="Screenshot 2019-10-14 at 10 41 20" src="https://user-images.githubusercontent.com/2526/66743203-4d78fa80-ee71-11e9-8012-d0f0362d5dd3.png">

Reply:
Some options:

 - `// @ts-ignore` this warning
 - remove the component name from the deprecation message
 - modify the `VersionedReference` interface (we likely don't want to do this)

I'll skip the warning for now and work on getting an otherwise green PR

Reply:
I think you can probably pass the `component` to the constructor when instantiating `StyleBindingReference` just above in `AttributeBinding.install` to avoid the issue.

https://github.com/emberjs/ember.js/blob/8f0e9b16e78766632ebc4c0ddd2f4ca7b3c9d4f4/packages/%40ember/-internals/glimmer/lib/utils/bindings.ts#L111

Reply:
ah yes, thanks!